client/state/notices/actions.js

```diff
@@ -38,6 +41,13 @@ function createNotice( status, text, options = {} ) {
 	}
 }
 
+export function setRoute( path ) {
```

Comment:
This doesn't appear to be used anywhere? Also, seems unrelated to notices, so while it might make sense for the notices reducer to hook in to the action type, I think the action should probably live elsewhere (in a potential routing subtree, perhaps?).